File: app/src/main/java/com/neome/api/meta/base/dto/StudioEntDeployVarMap.java
// neome.ai API - do not change
//

package com.neome.api.meta.base.dto;

import com.neome.api.meta.base.Types.MetaIdVar;

import java.util.Map;

@SuppressWarnings("unused")
public class StudioEntDeployVarMap extends StudioBase
{
  public MetaIdVar[] keys;

  public Map<MetaIdVar, StudioEntDeployVar> map;
}

File: app/src/main/java/com/neome/api/meta/base/dto/StudioEntDeployVarMap.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import java.util.Map
import com.neome.api.meta.base.Types.MetaIdVar
import com.neome.api.meta.base.dto.StudioBase
import com.neome.api.meta.base.dto.StudioEntDeployVar

interface StudioEntDeployVarMap : StudioBase
{
  val keys: Array<MetaIdVar>
  val map: Map<MetaIdVar, StudioEntDeployVar>
}